src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  margin: 0;
+  height: 96px;
```

Comment:
It's a bad solution, don't set height to body (what about other content of page? how can page have 96px height?)

Reply:
And avoid height hardcoding if you don't need it, let the content to dictate height of it container